public/index.html

```diff
@@ -209,6 +223,10 @@ <h2>TIL</h2>
           <h2>TheZaurus</h2>
         </header>
         <a target="_blank" href="https://thezaurus.znk.io"> <img src="img/zenika_logo_64.png" alt="thezaurus" /> </a>
+        <footer>
+          <a target="_blank" href="https://github.com/zenika-open-source/thezaurus">
+            <img src="https://github.com/favicon.ico" alt="Source Code: Thezaurus" />
+          </a>
```

Comment:
Il manque la balise fermante pour l'élément footer.
```suggestion
        <footer>
          <a target="_blank" href="https://github.com/zenika-open-source/thezaurus">
            <img src="https://github.com/favicon.ico" alt="Source Code: Thezaurus" />
          </a>
        </footer>
```